no-browser-site-scrapers/UMassMemorial/index.js

```diff
@@ -11,9 +11,10 @@ module.exports = async function GetAvailableAppointments() {
     const timestamp = new Date();
 
     for (const site of sites) {
+        const { departmentID, ...restSite } = site;
         results.push({
             ...webData[site.departmentID],
```

Comment:
You could just use `departmentID` since you extracted it that would give it a purpose.